inference-engine/tests/functional/inference_engine/serialization/single_layer/scatter_update.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+#include <ngraph/opsets/opset3.hpp>
+
+#include "shared_test_classes/single_layer/scatter_update.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace LayerTestsDefinitions;
+using namespace ngraph::opset3;
```

Comment:
Can we use opset8?

Reply:
But why :D ? ScatterUpdate was introduced with `opset3`